common/src/main/java/org/mvndaemon/mvnd/common/logging/TerminalOutput.java

```diff
@@ -173,6 +173,11 @@ private boolean doAccept(Message entry) {
             this.maxThreads = bs.getMaxThreads();
             final int maxThreadsDigits = (int) (Math.log10(maxThreads) + 1);
             this.threadsFormat = "%" + (maxThreadsDigits * 3 + 2) + "s";
+            if (maxThreads == 1) {
```

Comment:
Should we use a configuration flag for that ?

Reply:
You mean some users might not want this auto-rolling? I must say I like it like this. I would not want to add the flag unless somebody explicitly asks for it. Of course, if you personally want this, I'll add it immediately.

Reply:
I'm fine with the auto-rolling behaviour.  Let's advertise the change in the release notes and see if people want a switch.  I've read users saying they do really like the buffered output a lot, but for linear builds it may be a bit different.